### DISCUSSION TITLE
Render HTML components within a label

### DISCUSSION THREAD

**Discussion by dcworldwide** [kurkle/chartjs-chart-treemap#190] [Q&A]
Re: labelling. I'd like to render some a HTML component, instead of text. Is this possible?

My specific use case is react based, but it's sufficient to render html. The html is an user avatar (image + name + link) 

E.g.

```
labels: {
                            align: 'left',
                            display: true,
                            formatter(ctx) {
                                if (ctx.type !== 'data') {
                                    return 'transparent'
                                }
                                const item = ctx.raw
                                return [<MyComponent args />]
                            },
                            position: 'top'
                        }
```

**Comment by fabricio-fn** [comment]
if you are using react try this to render a user profile component

![Screenshot_2023-08-07_10-50-23](https://github.com/kurkle/chartjs-chart-treemap/assets/116690430/5160a761-c428-4823-9b65-912dd12ab020)
![Screenshot_2023-08-07_10-50-50](https://github.com/kurkle/chartjs-chart-treemap/assets/116690430/872fe724-906a-4071-8720-a122a18ed4e9)
